pkg/operator/controller/controller_test.go

```diff
@@ -313,7 +312,7 @@ var _ = Describe("Controller", func() {
 					},
 				}
 
-				Expect(rule.Spec.Groups[0].Rules).To(ContainElement(cdiDownAlert))
+				Expect(rule.Spec.Groups[1].Rules).To(ContainElement(cdiDownAlert))
```

Comment:
rookie question here, having a split to groups between recording&alerting is cleaner?

Reply:
Yes it is needed and recommended